hedera-node/docs/design/services/token-service/airdrops/airdrop-transaction.md

```diff
@@ -0,0 +1,208 @@
+
+              - It’s not existing and the AccountID is an `evm_address` →
+                the airdrop would first lazily-create a new account with `maxAutoAssociations` field of -1 and directly transfer the token being airdropped to it. This would include a `TokenTransferList` and the alias of the newly created account in the `TransactionRecord`
+    - Fee calculation logic
+        - assess all required fees for the airdrop from the payer of the transaction. This includes:
+            - airdrop fee (assessed always)
+            - first auto-renew fee for the token (assessed always)
+            - custom token fees covered by the sender (assessed always if present)
+            - crypto transfer fee (pre-paid always)
+            - token association fee (pre-paid if account is not associated to the token)
+            - auto account creation (assessed if the recipient does not exist and they are referred by their public `ECDSA` key or `evm_address`)
+- Interact with the pending airdrop state - if the recipient is not associated to the token, then the `TokenAirdropTransaction` responsibilities in term of interacting with the pending state would be to just add the airdrop information in the state
+- Add this new operation type to the `ThroughputLimits` throttle bucket group, so that it's included in the throttling mechanism
+
+## Acceptance Tests
+
+* Verify that an airdrop with a token and existing recipient with `maxAutoAssociations=-1` works correctly and the recipient receives the token directly
+* Verify that an airdrop with a token and existing recipient with `maxAutoAssociations` equal to a positive number and there are free `autoAssociations` slots works correctly and the recipient receives the token directly
+* Verify that an airdrop with a token and existing recipient with `maxAutoAssociations=0` and missing token association, works correctly and the airdrop lends in the pending state
+* Verify that an airdrop with a token and existing recipient with `maxAutoAssociations` equal to a positive number and there are no free `autoAssociations` slots works correctly and airdrop lends in the pending state
+* Verify that an airdrop with a token and a missing recipient pointed by its public `ECDSA key alias`, works correctly by auto-creating the recipient with `maxAutoAssociations=-1` and the token is directly transferred to it
+* Verify that an airdrop with a token and a missing recipient pointed by its `evm_address` alias, works correctly by lazy-creating the recipient with `maxAutoAssociations=-1` and the token is directly transferred to it
+* Verify that an airdrop with a missing TokenID fails
+* Verify that an airdrop with a missing recipient pointed by an `ED25519 key alias` fails
```

Comment:
Are we certain this is not supposed to work?  As far as I know it is entirely valid to send a `CryptoTransfer` to an ED25519 public key and it will auto-create the account, setting the `alias` to the public key bytes.

Reply:
You're right. That's still a valid case. I removed this scenario.